### DISCUSSION TITLE
Slightly modify font size to get rid of space on the right/bottom

### DISCUSSION THREAD

**Discussion by simonmandlik** [wezterm/wezterm#4214] [Ideas]
When the height/width of the window is not perfectly divisible by cell height/width, some "fractional" space is left there.

This is quite aesthetically unpleasing when used together with static text on the bottom (e.g. tmux line).

Given how seamlessly wezterm supports fractional font sizes and cell sizes, I was wondering if it would be possible to get rid of the space on the bottom / on the right by "distributing" it equally to the existing cells.

This would mean that the actual font size and / or cell size would very slightly grow from what is specified in the config (depending on the total number of cells that fit in the window), but I would be perfectly fine with that.

**Comment by wez** [comment]
TL;DR: No.
Longer answer: please search existing issues for a longer discussion.
